snmp/datadog_checks/snmp/data/default_profiles/netgear-readynas.yaml

```diff
@@ -0,0 +1,126 @@
+extends:
+  - _generic-ucd.yaml
+  - netgear-generic.yaml
+sysobjectid:
+  - 1.3.6.1.4.1.4526.100.12.*
+  - 1.3.6.1.4.1.4526.100.15.*
+  - 1.3.6.1.4.1.4526.100.16.*
+  - 1.3.6.1.4.1.4526.22.*
+metrics:
+  - MIB: READYNASOS-MIB
+    symbol:
+      OID: 1.3.6.1.4.1.4526.22.1.0
+      name: netgear.nasMgrSoftwareVersion
+      tag: netgear_nas_mgr_software_version
```

Comment:
This is a scalar string metric and we don't support this for now. This type of metric is dropped which is why the assertion doesn't find a match and [test](https://github.com/DataDog/integrations-core/pull/15322/files#diff-b31147c990110c49296e7e6f3b493de8b67d261e889314924b3c94fe4e02b404R40) fails
```suggestion
#  - MIB: READYNASOS-MIB
#    symbol:
#      OID: 1.3.6.1.4.1.4526.22.1.0
#      name: netgear.nasMgrSoftwareVersion
#      tag: netgear_nas_mgr_software_version
#  scalar string metric is not supported yet (keep this metric and this comment in profile until it's fixed)
```